pyvista/core/utilities/geometric_objects.py

```diff
@@ -363,6 +364,353 @@ def Sphere(
     return surf
 
 
+def SolidSphere(
```

Comment:
I think we explicitly have to add this to `doc/source/api/utilities/geometric.rst`, current doc build artifact doesn't contain this "class".

Reply:
Thanks for catching this. Done.